### PR TITLE
fix: [PIE-2986]: pipeline-grey-border variable added

### PR DESCRIPTION
**Pull request by PratyushGar**
##### Summary:

css variable added for canvasWidget to access

##### Jira Links:

[https://harness.atlassian.net/browse/PIE-2986](https://harness.atlassian.net/browse/PIE-2986)

##### Screenshots:
**Updated:**
Screenshot 2022-03-09 at 1 05 22 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by PratyushGar** [comment]
retrigger titlecheck

**Review by yc147 — APPROVED** [review]

**Review by AshwinBhatkal-Harness — DISMISSED** [review]

File: src/modules/70-pipeline/components/Diagram/canvas/CanvasWidget.module.scss [hunk 1]
```diff
@@ -12,6 +12,7 @@
   --pipeline-subtext: var(--grey-300);
   --pipeline-barrier: var(--yellow-500);
   --pipeline-barrier-yellow: var(--yellow-400);
+  --pipeline-grey-border: #c8c6c6;
```
Comment (AshwinBhatkal-Harness) [inline src/modules/70-pipeline/components/Diagram/canvas/CanvasWidget.module.scss:15]: Was this color given by UX?

Comment (PratyushGar) [reply src/modules/70-pipeline/components/Diagram/canvas/CanvasWidget.module.scss:15]: This is already being used in the pipeline execution - same functionality/

**Review by ishantmehtaharness — APPROVED** [review]

File: src/modules/70-pipeline/components/Diagram/canvas/CanvasWidget.module.scss (hunk 1 above)
Comment (ishantmehtaharness) [reply src/modules/70-pipeline/components/Diagram/canvas/CanvasWidget.module.scss:15]: Can we instead use/create variable for `#c8c6c6` ?

**Review by ishantmehtaharness — CHANGES_REQUESTED** [review]
https://github.com/harness/harness-core-ui/pull/7742/files#r822366024

**Review by ishantmehtaharness — APPROVED** [review]
Approved as per @yc147

File: src/modules/70-pipeline/components/Diagram/canvas/CanvasWidget.module.scss (hunk 1 above)
Comment (yc147) [reply src/modules/70-pipeline/components/Diagram/canvas/CanvasWidget.module.scss:15]: @PratyushGar Talk to UX folks and get this color defined the uicore or use the one which already exists in uicore.

**Review by yc147 — CHANGES_REQUESTED** [review]
Talk to UX folks and get this color defined the uicore or use the one which already exists in uicore.

**Review by ishantmehtaharness — CHANGES_REQUESTED** [review]
https://github.com/harness/harness-core-ui/pull/7742/files#r822366024

**Review by yc147 — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Review by ishantmehtaharness — APPROVED** [review]
Looks good. Thanks for making the changes

**Comment by PratyushGar** [comment]
trigger cypress

**Pull request merged**